$NetBSD: patch-Makefile,v 1.3 2023/10/11 03:41:01 ryoon Exp $

* Disable symbol comparison, it's fragile
* Workaround for rebuild in install stage.

--- Makefile.orig	2023-10-07 00:10:58.000000000 +0000
+++ Makefile
@@ -790,9 +790,9 @@ check-symbols: startup_files libc
 
 	# Check that the computed metadata matches the expected metadata.
 	# This ignores whitespace because on Windows the output has CRLF line endings.
-	diff -wur "expected/$(TARGET_TRIPLE)" "$(SYSROOT_SHARE)"
+	#diff -wur "expected/$(TARGET_TRIPLE)" "$(SYSROOT_SHARE)"
 
-install: finish
+install:
 	mkdir -p "$(INSTALL_DIR)"
 	cp -r "$(SYSROOT)/lib" "$(SYSROOT)/share" "$(SYSROOT)/include" "$(INSTALL_DIR)"
 
